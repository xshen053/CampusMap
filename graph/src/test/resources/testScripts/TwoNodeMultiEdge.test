# Tests adding two nodes, multiple edges to graph.

# Create a graph
CreateGraph TNME_graph

# Add two nodes
AddNode TNME_graph a
AddNode TNME_graph b

# Add five edges
AddEdge TNME_graph a a AA
AddEdge TNME_graph a b AB1
AddEdge TNME_graph a b AB2
AddEdge TNME_graph b a BA
AddEdge TNME_graph b b BB

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
ListNodes TNME_graph
ListChildren TNME_graph a
ListChildren TNME_graph b