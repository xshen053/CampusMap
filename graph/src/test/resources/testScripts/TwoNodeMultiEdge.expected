# Tests adding two nodes, multiple edges to graph.

# Create a graph
created graph TNME_graph

# Add two nodes
added node a to TNME_graph
added node b to TNME_graph

# Add five edges
added edge AA from a to a in TNME_graph
added edge AB1 from a to b in TNME_graph
added edge AB2 from a to b in TNME_graph
added edge BA from b to a in TNME_graph
added edge BB from b to b in TNME_graph

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
TNME_graph contains: a b
the children of a in TNME_graph are: a(AA) b(AB1) b(AB2)
the children of b in TNME_graph are: a(BA) b(BB)